akka-http-core/src/main/java/akka/http/javadsl/model/headers/HttpOriginRange.java

```diff
@@ -10,10 +10,10 @@
 /**
  * @see HttpOriginRanges for convenience access to often used values.
  */
-public abstract class HttpOriginRange {
-    public abstract boolean matches(HttpOrigin origin);
+public interface HttpOriginRange {
```

Comment:
ok, but I think changing a method from an AbstractClass to an Interface may cause runtime errors from just calling those (not 100% about this)